lib/ansible/module_utils/facts/system/distribution.py

```diff
@@ -325,6 +325,11 @@ def parse_distribution_file_Debian(self, name, data, path, collected_facts):
             release = re.search('DISTRIB_RELEASE=(.*)', data)
             if release:
                 debian_facts['distribution_release'] = release.groups()[0]
+        elif path == '/etc/lsb-release' and 'Parrot' in data:
+            debian_facts['distribution'] = 'Parrot'
+            release = re.search('DISTRIB_RELEASE=(.*)', data)
+            if release:
+                debian_facts['distribution_release'] = release.groups()[0]
```

Comment:
Since this is duplicated code, I wonder if it could be consolidated with the `Kali` branch.

Reply:
Right. Done.